deploy/cw-operator.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: v1
+data:
+  controller_manager_config.yaml: |
+    apiVersion: controller-runtime.sigs.k8s.io/v1alpha1
+    kind: ControllerManagerConfig
+    health:
+      healthProbeBindAddress: :8081
+    metrics:
+      bindAddress: 127.0.0.1:8080
+    webhook:
+      port: 9443
+    leaderElection:
+      leaderElect: true
+      resourceName: 08db2feb.percona.com
+kind: ConfigMap
```

Comment:
do we need this only for CW or should we put it to `deploy/operator.yaml` too?

Reply:
@egegunes but it is in the operator.yaml as well.